test/test_jit.py

```diff
@@ -3639,6 +3808,14 @@ def forward(self, x, y):
     'test_clamp_max_scalar',
     'test_clamp_min',
     'test_clamp_min_scalar',
+    'test_std_dim',
+    'test_std_dim_1d',
+    'test_std_dim_1d_neg0',
+    'test_std_dim_neg0',
+    'test_var_dim',
+    'test_var_dim_1d',
+    'test_var_dim_1d_neg0',
+    'test_var_dim_neg0',
```

Comment:
Why did those start to fail now?

Reply:
I think what's going is the following:

there are two schemas for `var`: (* means that after this point everything is a kwarg).
(1) var(Tensor, int, *, bool, bool, Tensor)
(2) var(Tensor, *, bool)

(1) used to come before (2) in aten_schema.cpp.

I made the following changes to schemas:
- Shorter schemas come before longer schemas
- bool, int are treated as IntType rather than DynamicType like before

So the schemas look like the following in aten_schema:
(2) var(DynamicType, IntType)
(1) var(DynamicType, IntType, IntType, DynamicType)
Now, when you call `torch.var(tensor, dim=1)`, the compiler mistakingly matches it with (2) instead of (1), which is a problem.

I'll put a comment in the code with this (and think about if I can fix this quickly or not)